src/DIRAC/Interfaces/scripts/dstat.py

```diff
@@ -226,15 +227,15 @@ def main():
         )
 
         if not result["OK"]:
-            print("Error:", result["Message"])
+            gLogger.error("Error:", result["Message"])
             DIRACExit(-1)
 
         jobs = result["Value"]
 
     try:
         jobs = [int(job) for job in jobs]
     except Exception as x:
```

Comment:
A bit out of the scope of this PR: I think it is generally safer to have specific `except` clauses (instead of one general `except Exception` clause).
You probably have no idea of which exceptions to catch in most of these commands, but in this specific case for instance, you can have the following exceptions:

```suggestion
    except TypeError, ValueError as x:
```

Reply:
Yes, it's not my code and I wouldn't have written it that way in the first place, but I think this is stuff that will get picked up when people actually start using it, so I would leave it out of this PR.